components/common/layout.tsx

```diff
@@ -2,10 +2,16 @@ import React, { FC } from 'react';
 import Head from 'next/head';
 import Navbar from './navbar/navbar';
 
-const Layout: FC = ({ children }) => {
+interface LayoutProps {
+    pageName: string;
+}
+
+const Layout: FC<LayoutProps> = ({ children, pageName }) => {
+    const title = `Jerry's ${pageName}`;
```

Comment:
Extracted this to a variable to prevent the formatting rules from breaking out the <title> block into 4 lines. Addressing these gotcha's with the current linting rules is part of https://github.com/jarekb84/my/issues/31